lib/3scale/backend/storage_async/methods.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module Backend
+    module StorageAsync
+      module Methods
+        # Now we are going to define the methods to run redis commands
+        # following the interface of the redis-rb lib.
+        #
+        # These are the different cases:
+        #   1) Methods that can be called directly. For example SET:
+        #      call('SET', some_key)
+        #   2) Methods that need to be "boolified". These are methods for which
+        #      redis-rb returns a boolean, but redis just returns an integer.
+        #      For example, Redis returns 0 or 1 for the EXISTS command, but
+        #      redis-rb transforms that into a boolean.
+        #   3) There are a few methods that need to be treated differently and
+        #      do not fit in any of the previous categories. For example, SSCAN
+        #      which accepts a hash of options in redis-rb.
+        #
+        # All of this might be simplified a bit in the future using the
+        # "methods" in async-redis
+        # https://github.com/socketry/async-redis/tree/master/lib/async/redis/methods
+        # but there are some commands missing, so for now, that's not an option.
```

Comment:
seems like these were removed and likely no plan to bring them back, perhaps remove that comment

Reply:
The comment is outdated. `redis-async` now relies on https://github.com/socketry/protocol-redis to implement its methods. And if I'm not wrong, we could use that gem for `redis-rb` too. I'll take a look at some point in the future to make this simpler.

For now, I updated the comment on 054d0053e98e0a3fa82a99079867e7eba5446a35